tests/fixtures/netex_france/lignes.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PublicationDelivery version="1.09:FR-NETEX_LIGNE-2.1-1.0" xmlns="http://www.netex.org.uk/netex" xmlns:core="http://www.govtalk.gov.uk/core" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:ifopt="http://www.ifopt.org.uk/ifopt" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.netex.org.uk/netex">
+	<PublicationTimestamp>2019-04-03T17:19:00</PublicationTimestamp>
+	<ParticipantRef>Participant</ParticipantRef>
+	<dataObjects>
+		<CompositeFrame>
+			<frames version="any">
+				<ServiceFrame version="any">
+					<Network id="FR:network:TGN:" version="any">
+						<Name>The Great Network</Name>
+						<LineRef ref="FR:line:M1">
```

Comment:
Networks needs a members section to contain LineRef sections

Reply:
Done in d335dfc16a98add274bbd87ae20d07722f2d7838.